packages/mobile/src/components/CodeInput.tsx

```diff
@@ -105,22 +132,34 @@ export default function CodeInput({
                 // on the native input. Though it doesn't work in all cases (see https://stackoverflow.com/a/33227237/158525)
                 // and has the unfortunate drawback of breaking multiline autosize.
                 // We use numberOfLines to workaround this last problem.
-                keyboardType={Platform.OS === 'android' ? 'visible-password' : undefined}
+                keyboardType={
+                  Platform.OS === 'android'
+                    ? 'visible-password'
+                    : shortVerificationCodesEnabled
+                    ? 'number-pad'
+                    : undefined
```

Comment:
Don't we want `number-pad` too on Android?

Reply:
I'm slightly worried about the comment above `autoCorrect={false}`, which starts with `// This disables keyboard suggestions on iOS, but unfortunately NOT on Android ...`. So, I'm hesitant to replace`visible-password` with `number-pad`. WDYT?

Reply:
Yes I was fighting to disable `autoCorrect` on Android.
But here using `number-pad` should be safe on both iOS and Android when `shortVerificationCodesEnabled` is `true`.
And we can keep the existing logic when it's `false`.